test/unit/service/point-of-sale-service.ts

```diff
@@ -252,7 +252,7 @@ describe('PointOfSaleService', async (): Promise<void> => {
         useAuthentication: true,
       };
 
-      const response = await PointOfSaleService.directPointOfSaleUpdate(update);
+      const response = (await PointOfSaleService.updatePointOfSale(update)) as PointOfSaleWithContainersResponse;
```

Comment:
I think the update function should have a default response (with or without containers/products) such that the responses are also always consistent.